{"name": "test_firstTest", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "self = <test_myFirstTest.TestMush object at 0x106184770>\n\n    def test_firstTest(self):\n        print('First test1')\n        print('Second test')\n>       assert False\nE       assert False\n\nsrc/test_myFirstTest.py:5: AssertionError"}, "attachments": [{"name": "stdout", "source": "75406180-273d-4e94-9c02-417deda33b57-attachment.txt", "type": "text/plain"}], "start": 1745046092584, "stop": 1745046092584, "uuid": "6d7e6176-10ed-48b6-beef-7bbd38b72784", "historyId": "98b9d2a354c1b40711ae7fd21ed656bb", "testCaseId": "98b9d2a354c1b40711ae7fd21ed656bb", "fullName": "src.test_myFirstTest.TestMush#test_firstTest", "labels": [{"name": "parentSuite", "value": "src"}, {"name": "suite", "value": "test_myFirstTest"}, {"name": "subSuite", "value": "TestMush"}, {"name": "host", "value": "Musheer-Khan.local"}, {"name": "thread", "value": "54738-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.test_myFirstTest"}]}